scripts/PNG.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+png_ver=1.6.37
+
+download () {
+	get_tar_archive libpng "https://download.sourceforge.net/libpng/libpng-${png_ver}.tar.gz"
+}
+
+build () {
+	mkdir -p libpng
+	pushd libpng
+	$srcdir/libpng/configure --host=$CROSS_PREFIX
+	make && make DESTDIR=$PWD install
+	popd
+
+	local prefix=$PWD/libpng/usr/local
+	mkdir $pkgdir/include
+	cp -p $prefix/lib/libpng.a $pkgdir/
+	cp -p $prefix/include/*.h $pkgdir/include/
+}
```

Comment:
it was only like this to build it in the same folder as irrlicht, you should be able to simplify it to this:
```suggestion
build () {
	$srcdir/libpng/configure --host=$CROSS_PREFIX
	make
	make_install_copy
}
```

Reply:
Hmm, maybe. Need to find my dusty Android build environment to check... (the code itself is month old, I just didn’t make a PR for some reason).

Reply:
Done